playground/twindleco/header/header7/header7.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
   <header>
-    <h3 id='logo'>Twindle-logo</h3>
+    <h3 id='logo'><img src ="C:\Users\Eudy\Documents\GitHub\twindle\playground\twindleco\Twindle-web-Teampage\images\twindle.png" alt="Twindle-logo"width="100px"height="100px"></h3>
```

Comment:
@Na-ilah Instead of specifying system path, use relative path and verify the changes. System path will not work on other dev's machines.

Reply:
Oh thank you for the specific help.. how do I do the relative path? I am new at this

Reply:
Check the VSCode options screenshot,

![image](https://user-images.githubusercontent.com/72256599/96904031-358c7b80-14b4-11eb-8544-fc0e71602f14.png)

Hope this will help you!

Reply:
Check the below VSCode example file options screenshot,

![image](https://user-images.githubusercontent.com/72256599/96904609-fa3e7c80-14b4-11eb-8b36-2806893708c3.png)

Hope this will help you!

Reply:
@ni3-kdm I think I did it. Thanks for your help